app/views/layouts/application.html.erb

```diff
@@ -1,3 +1,35 @@
 <%= render layout: "layouts/application" do %>
-  <%= yield %>
+  <div class="govuk-width-container">
```

Comment:
Is this all just to get rid of the width-container?
If so, the following would be much simpler in _application:
```
<% if content_for?(:width_container) %>
  <%= yield(:width_container) %>
<% else %>
  <div class="govuk-width-container">
<% end %>
```
and something similar at the end to close the div

Reply:
I've pulled the width-container layout out into a new partial to simply wrapping the main content in a width container. what do you think to that?

Reply:
Looks good